app/views/patients/show.html.erb

```diff
@@ -126,10 +130,14 @@
           <td><%= cc.last_name %></td>
           <td><%= cc.primary_telephone.present? && cc.primary_telephone.length > 10 ? cc.primary_telephone&.gsub('+1', '')&.gsub(/\D/, '')&.insert(3, '-')&.insert(7, '-') : '' %></td>
           <td><%= cc.email %></td>
-          <td><%= cc.contact_attempts %></td>
+          <td class="text-center"><%= cc.last_date_of_exposure || 'Unknown' %></td>
```

Comment:
something I just noticed, we generally want to display dates as MM/DD/YYYY in the UI